src/cluster_legacy.c

```diff
@@ -6532,3 +6851,55 @@ int clusterAllowFailoverCmd(client *c) {
 void clusterPromoteSelfToMaster(void) {
     replicationUnsetMaster();
 }
+
+/* Replicate all migrating and importing slot states to a
+ * a given replica client. */
```

Comment:
Do we broadcast to all replica(s)? I don't see any replica client.

Reply:
this is on the full sync path. we only need to catch the replica up on the full sync path.

Reply:
So we replicate to all clients that need it then? Not "a given"

```suggestion
 * replica clients. */
```

Reply:
> Do we broadcast to all replica(s)? I don't see any replica client.

Take it back - you are right that this replicates to all replicas. Will update the comment accordingly.